proposed/http-handlers/request-handlers.md

```diff
@@ -52,6 +52,12 @@ the creation of a resulting response, as defined by PSR-7.
 A middleware component MAY create and return a response without delegating to
 a request handler, if sufficient conditions are met.
 
+When delegating from one middleware to the next in a sequence, a dispatching
+system should use an intermediary request handler as a way to link middleware
+together. The final or innermost middleware will act as a gateway to application
+code and generate a response from its results; alternately, the middleware may
+delegate this responsibility to a dedicated request handler.
```

Comment:
Nitpick: SHOULD and MAY should be capitalized to mean the defined SHOULD and MAY, I think, since this is in the spec proper.

Reply:
My understanding is they should only be capitalized when they have the semantic meanings as defined in RFC 2119; the question is whether or not those words here have those semantic meanings. I did not originally write this with those meanings in mind.

The problem, of course, is that there are very few ways to re-word this to have the same intended meaning _without_ using those words. Can you propose an alternative? Or do you feel they have those semantics?

Reply:
If not RFC2119, maybe `can` could be used for both?

Reply:
`CAN` also has semantic meaning via RFC 2119. :)

Been discussing this and the idea of moving the section to the spec document itself heavily today; hopefully we can have resolution shortly.

Reply:
Hmm, I don't see `CAN` mentioned in the RFC... hence me thinking it safe.

Would something wordy, like `could potentially`, do the job without sounding like a 2119 directive?